dev/com.ibm.ws.org.objectweb.asm/bnd.overrides

```diff
@@ -8,4 +8,5 @@ Implementation-Title: ASM all classes
 DynamicImport-Package: *
 
```

Comment:
A comment in bnd.bnd or bnd.overrides to instruct people to update the CURRENT_ASM field in ASMHelper if the version is changed to a new major version would be very helpful.  Somewhere near where the version is specified.